docs/architecture/posixfs.md

```diff
@@ -0,0 +1,173 @@
+2. Group based: All users who should be able to manipulate files have to be in a unix group. The Infinite Scale user has also to be in there. The default umask in the directory used has to allow group writing all over the place.
+3. Impersonation: Infinite Scale impersonates the user who owns the folder on the file system to mimic the access as the user.
+
+All possibilities have pros and cons for operations.
+
+One for all, it seems reasonable to use LDAP to manage users which is the base for the Infinite Scale IDP as well as the system login system via PAM.
+
+## Advanced Features
+
+Depending on the capabilities of the underlying file system, the Infinite Scale PosixFS can benefit from more advanced funcitonality described here.
+
+### Versioning
+
+If the underlying file system is able to create versions of single resources (imagine a git based file system) this functionality could directly be used by Infinite Scale.
+
+In the current state of the PosixFS, versioning is not supported.
+
+### Trashbin
+
+If the underlying file system handles deleted files in a trash bin that allows restoring of previously removed files, this functionality could directly be used by Infinite Scale.
+
+If not available it will follow the [the Free Desktop Trash specificaton](https://specifications.freedesktop.org/trash-spec/trashspec-latest.html).
+
+In the current state of the PosixFS, trash bin is not supported.
+
+## Limitations
+
+As of Q2/2024 the PosixFS is in technical preview state which means that it is not officially supported.
+
+The tech preview comes with the following limitations:
+
+1. User Management: Manipulations in the file system have to be done by the same user that runs Infinte Scale
+2. Only inotify and GPFS file system change notification methods are supported
+3. Advanced features versioning and trashbin are not supported yet
+4. The space/project folders in the filesystem are named after the UUID, not the real space name
+5. No CephFS support yet
+
+## Setup
+
+This describes the steps to use the PosixFS storage driver with Infinite Scale.
+
+It is possible to use different storage drivers in the same Infinite Scale installation. For example it is possible to set up one space running on PosixFS while others run decomposedFS.
+
+### Prerequisites
+
+To run PosixFS, the following prerequisites have to be fulfilled:
+
+1. There must be storage available to store meta data and blobs, available under a root path
+2. When using inotify, the storage must be local on the same machine. Network mounts do not work with Inotify
+3. The storage root path must be writeable and executable by the same user Infinite Scale is running under
+4. An appropiate version of Infinte Scale is installed, version number 5.0.5 and later
+5. Either redis or nats-js-kv cache service
```

Comment:
Please don't mention `redis`. It is not working properly. We want to use `nats-js-kv` as only suggested distributed cache solution.